core/src/edu/rice/comp413fall2020grey/Location/interest/EqualityStringPredicate.java

```diff
@@ -0,0 +1,19 @@
+package edu.rice.comp413fall2020grey.Location.interest;
+
+import com.mongodb.client.model.Filters;
+import org.bson.conversions.Bson;
+
+import java.io.Serializable;
+import java.util.HashMap;
+
+public class EqualityStringPredicate extends EqualityPredicate<String>  {
+
+    public EqualityStringPredicate(String field, String value) {
```

Comment:
There isn't a relative option for string predicates?

Reply:
What would be the use case for this?